gradle/dependency-locks/default.lockfile

```diff
@@ -24,8 +25,9 @@ net.bytebuddy:byte-buddy:1.7.9
 org.apache.commons:commons-compress:1.9
 org.apache.commons:commons-csv:1.3
 org.apache.commons:commons-exec:1.3
-org.apache.commons:commons-lang3:3.5
+org.apache.commons:commons-lang3:3.9
```

Comment:
Is `RandomUtils` worth importing a lib for? It's just a bit of syntax sugar.
Adding this dependency in `random-data` brings unnecessary version conflicts.

Reply:
It's transitional from commons-text. I originally used Kotlin Random, but that comes from Kotlin lib 1.3... Anyway, I got rid of all these dependencies now.

Reply:
Thanks.
PS. So far `java.util.Random` worked fine, no need for Kotlin or Apache wrappers.